common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -818,6 +822,8 @@ def _get_answer_notification(self, render_notifications):
                     ).format(progress=str(progress))
                 else:
                     answer_notification_message = _('Partially Correct')
+            elif answer_notification_type == 'answered':
+                answer_notification_message = self.submitted_message
```

Comment:
@pomegranited I think it might make sense to consolidate the "answered"/"submitted" terminology. That is, change `'answered'` to `'submitted'` or rename `submitted_message` to `answered_message`. I'd prefer the former. (But maybe the latter would make more sense in the general context of the implementation?)